keyext.slicing/src/main/java/org/key_project/slicing/graph/EquivalenceDirectedGraph.java

```diff
@@ -67,4 +67,15 @@ public Collection<GraphNode> getVerticesModProofIrrelevancy(GraphNode v) {
             return List.of(v);
         }
     }
+
+    public EquivalenceDirectedGraph copy() {
```

Comment:
I think it would be a good idea to override the clone() method here, since it is allowed to refine the return type covariantly.

Reply:
clone() has `protected` access, it would not work here.